src/components/textarea.js

```diff
@@ -36,6 +36,9 @@ module.exports = function(app) {
             }
             return 'formio/components/texteditor.html';
           }
+          else if (!$scope.readOnly && $scope.component.components) {
```

Comment:
Hi Travis,

I don't think it applies to all types of components because the code occurs in .../textarea.js and should only affect textarea components which don't normally have sub-components.  It's for the action condition case.  See pull request at formio.